packages/ui/src/composables/useElementBackground.ts

```diff
@@ -0,0 +1,149 @@
+import { Ref, ref, watchEffect } from 'vue'
```

Comment:
Would be nice to have story or unit test to support "optimized" claim. So that we can be sure it's fast enough.

Reply:
How can we test if it optimized if everyone has a different PC specs?

I think we all just need to believe that this way is faster than parsing all of the CSS on a page and check if it has CSS variable and search for CSS variable again as it was previously.

Reply:
@m0ksem CI has constant spec.

+ for optimization you probably mean something like 50 times faster? Not 5% faster right? So would be noticeable regardless of spec.

But I agree with your point. Let's not press that.

Reply:
I'm not sure how much performant it is, but I just removed large operations, which was a bottleneck. Also, this solution is much more stable and have no edge cases (at least, I didn't find them)